commons-numbers-complex/src/test/java/org/apache/commons/numbers/complex/ComplexTest.java

```diff
@@ -1378,6 +1425,32 @@ private static void assertSignedZeroArithmetic(String name, BiFunction<Complex,
         }
     }
 
+    private static void assertSignedZeroArithmetic(String name, BiFunction<Complex, Double, Complex> doubleOperation1,
```

Comment:
Duplicating this method is not as maintainable as having only one version of it. I have suggested a call for the `subtractFromReal` and `subtractFromImaginary` tests to use a lambda for ComplexFunctions. You can then have only one instance of this method.

Can you indent the arguments 8 spaces (from the `private` keyword).